test/MarketMaker.js

```diff
@@ -174,7 +174,7 @@ contract('Market validation Mechanism', function(accounts) {
 
     assert.equal(data.MVMMonth >= periods, await mm.isFinished());
 
-    data.ethBalances[customerAddressIndex].should.be.bignumber.equal(web3.eth.getBalance(customer));
+    // data.ethBalances[customerAddressIndex].should.be.bignumber.equal(web3.eth.getBalance(customer));
```

Comment:
is this line still failing, or can we uncomment it already?

Reply:
:+1:

Reply:
I think we are doing something wrong now on the eth balance calculation on the test once the MVM buy the tokens back, the test fail when now it takes the 25% extra tokens in count.